pydantic/read_env_file.py

```diff
@@ -0,0 +1,23 @@
+import re
```

Comment:
I don't think this needs a separate file, please move to the bottom of `env_settings.py`.